feathr_project/feathr/version.py

```diff
@@ -1 +1,10 @@
-__version__ = "0.9.0-rc2"
\ No newline at end of file
+__version__ = "0.9.0-rc2"
+
+def get_version():
+    return __version__
+
+# Decouple Feathr MAVEN Version from Feathr Python SDK Version
+import os
+def get_maven_artifact():
```

Comment:
Might be better if we call this `get_maven_artifact_version`?

Reply:
This method doesn't return the version name like `0.8.0` or `0.9.0-rc1` but the whole artifact name like `com.linkedin.feathr:feathr_2.12:0.8.0` for spark provider to use. I think `maven_artifact` would be more accurate.